madmom/audio/spectrogram.py

```diff
@@ -14,7 +14,10 @@
 from ..processors import Processor, SequentialProcessor
 from .stft import PropertyMixin
 from .filters import (LogarithmicFilterbank, NUM_BANDS, FMIN, FMAX, A4,
-                      NORM_FILTERS, UNIQUE_FILTERS)
+                      NORM_FILTERS, UNIQUE_FILTERS,
+                      SemitoneBandpassFilterbank)
```

Comment:
please reorder them to: classes, constants. While you're at it, they could be re-ordered alphabetically.